src/sdk/cpp/lib/CMakeLists.txt

```diff
@@ -38,8 +38,12 @@ add_library(iotea_sdk_cpp
   SHARED
     src/util.cpp
     src/logging.cpp
-    src/iotea.cpp
+    src/event.cpp
```

Comment:
Is there an order? If not we could introduce one (alphabetical) but this is only a 'nice to have'

Reply:
Re-ordred.